### DISCUSSION TITLE
Libriparty Dataset

### DISCUSSION THREAD

**Discussion by syoon-asapp** [speechbrain/speechbrain#1611] [General]

**Comment by Adel-Moumen** [comment]
?

**Comment by jonnor** [comment]
What is the LibriParty dataset? It is mentioned in https://huggingface.co/speechbrain/vad-crdnn-libriparty - but apart from this issue I cannot find any other references to?

**Reply by jonnor** [reply]
My google-fu must be getting weak. I found it now at, https://github.com/speechbrain/speechbrain/tree/develop/recipes/LibriParty/generate_dataset
